dart.dart

```diff
@@ -0,0 +1,17 @@
+void fizzBuzz(int foo) {
+  if (foo % 15 == 0) {
+    print("FizzBuzz");
+  } else if (foo % 5 == 0) {
+    print("Buzz");
+  } else if (foo % 3 == 0) {
+    print("Fizz");
+  } else {
+    print(foo);
+  }
+}
+
+void main() {
+  for (var i = 1; i <= 100; i++) {
```

Comment:
It looks like when compiled to Javascript it doesn't actually matter, but I think it's probably important for understanding Dart's intentional semantics: https://api.dartlang.org/stable/1.20.1/dart-core/int-class.html

It probably also helps static analysis tools quite a lot.